src/Streamly.hs

```diff
@@ -162,8 +162,8 @@ module Streamly {-# DEPRECATED "Please use \"Streamly.Prelude\" instead." #-}
     , aheadly
     , wAsyncly
     , parallely
-    , zipSerially
-    , zipAsyncly
+    , fromZipSerial
+    , fromZipAsync
```

Comment:
In this module, we need to deprecate `zipSerially` and `zipAsyncly` instead of renaming them.